models/model.js

```diff
@@ -10,6 +10,20 @@ const fetchTopics = () => {
     .then(result => result.rows)    
 }
 
+const fetchArticles = () => {
+    return db.query(
+        `
+        SELECT articles.author, articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, articles.article_img_url, CAST(COUNT(comments.article_id) AS INT) as comment_count
+        FROM articles
+        LEFT JOIN comments ON articles.article_id = comments.article_id
+        GROUP BY articles.author, articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, articles.article_img_url
+        ORDER BY articles.created_at ASC;
```

Comment:
This should be DESCending instead of ASC, meaning that the earliest date will be served first.

The date is saved as seconds since a given time, so when you sort by descending, the earliest date comes first.

Reply:
hi modou ok now im confused. when i console log the response from the server i get this: `console.log
    [
      2020010, 2020011,
      2020041, 2020050,
      2020051, 2020060,
      2020070, 2020080,
      2020101, 2020101,
      2020101, 2020110
    ]`
is that not the expected order?